### PR TITLE
feat(starr): add DV HDR10Plus CF

### LINKED ISSUE #1285: [Bug] A file containing both HDR10+ and DV profiles is incorrectly matched
**Which Guide**
HDR10+ profile within the Radarr guide

**Describe the Error in the Guide**
The following file contains both DV and HDR10+, but the matching shows only [DV] and [Hybrid]
```
mediainfo Ant-Man.and.the.Wasp.Quantumania.2023.German.EAC3D.DL.2160p.Hybrid.WEB.DV.HDR10Plus.HEVC-QfG.mkv
...
HDR format: Dolby Vision, Version 1.0, dvhe.08.06, BL+RPU, HDR10 compatible / SMPTE ST 2094 App 4, Version 1, HDR10+ Profile A compatible
...
```
**Expected behavior**
The file should also match [HDR10+] and [HDR10+ Boost] and therefore be downloaded.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by nuxencs**
# Pull Request

## Purpose

Add a custom format for `DV HDR10Plus` that was just added to the Starrs.
Fix #1285 

## Approach

- [x] Add `DV HDR10Plus` CF
- [x] Update `DV HDR10` CF regex
- [x] Add `DV HDR10Plus` to the guides

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by bakerboy448 — APPROVED** [review]
Lgtm

File: docs/Radarr/Radarr-collection-of-custom-formats.md [hunk 1]
```diff
@@ -507,6 +508,26 @@ I also made 3 guides related to this one.
 
 ------
 
+### DV HDR10Plus
+
+<sub>Dolby Vision = DoVi = DV</sub>
+
+??? question "DV HDR10 - [Click to show/hide]"
```
Comment (yammes08) [inline docs/Radarr/Radarr-collection-of-custom-formats.md:515]: Looks like this needs updating - the expander title and description

File: docs/Sonarr/sonarr-collection-of-custom-formats.md [hunk 2]
```diff
@@ -523,6 +524,26 @@ I also made 3 guides related to this one.
 
 ------
 
+### DV HDR10Plus
+
+<sub>Dolby Vision = DoVi = DV</sub>
+
+??? question "DV HDR10 - [Click to show/hide]"
```
Comment (yammes08) [inline docs/Sonarr/sonarr-collection-of-custom-formats.md:531]: Looks like this needs updating - the expander title and description

**Review by yammes08 — COMMENTED** [review]
Two changes, otherwise all looks good.

**Review by yammes08 — CHANGES_REQUESTED** [review]
Changes as per comments

**Comment by nuxencs** [comment]
> Changes as per comments

its the same for all the DV formats 😄

**Comment by yammes08** [comment]
> > Changes as per comments
> 
> its the same for all the DV formats 😄

Ah 😄 - maybe leave it for now and handle it in the other PR for description cleanup.

**Pull request merged**